guidelines/responses.md

```diff
@@ -2,10 +2,16 @@
 
 ## Content-type
 
-The API responds with `Content-Type` set to `application/json` and JSON content. In case of success, the HTTP status code is 200 and the content contains result according to the call. In case of error, there are multiple HTTP status codes for different types of errors:
+The API responds with `Content-Type` set to `application/json` and JSON content in body. In case of 204 response, the `Content-Type` header is not set and response body is empty. 
```

Comment:
```suggestion
The API responds with `Content-Type` set to `application/json`, and with JSON content in the body. In case of a 204 response (see [Request minimal response](requests.md#request-minimal-response)), the `Content-Type` header is _not_ set and the response body is empty. 
```